App.js

```diff
@@ -4,38 +4,118 @@ function MainContainer() {
   return (
     <div>
       <colgroup className="MainContainer">
-        <SideBar />
+        <div>
+          <SideBar />
+          <UserButton />
+        </div>
         <NewsFeedCol />
         <TrendingCol />
       </colgroup>
     </div>
   );
 }
 
+function Buttons() {
```

Comment:
I think it would be good practice to create each button based on some date.

Here's what you should do:

1) Store the data that's used for each button into an array of objects:

```jsx
const buttons = [{ imageSource: '...', text: 'Home' }]
```

2) Then, `map` through each object in the `buttons` array and render the markup:

```jsx
buttons.map(buttonData => {
  return (
    <button className="Button">
        <img src={buttonDate.imageSource} />
        {buttonData.text && <p className="ButtonText">{buttonData.text}</p>}
    </button>
  )
})
```